examples/config.c

```diff
@@ -0,0 +1,304 @@
+#ifndef _POSIX_C_SOURCE
+#define _POSIX_C_SOURCE 200809L
+#endif
+#include <stdlib.h>
+#include <limits.h>
+#include <getopt.h>
+#include <string.h>
+#include <unistd.h>
+#include <wlr/util/log.h>
+#include <wlr/types/wlr_geometry.h>
+#include "shared.h"
+#include "config.h"
+#include "ini.h"
+
+static void usage(const char *name, int ret) {
+	fprintf(stderr,
+		"usage: %s [-C <FILE>]\n"
+		"\n"
+		" -C <FILE>      Path to the configuration file (default: wlr-example.ini).\n"
+		"                See `examples/wlr-example.ini.example` for config file documentation.\n", name);
+
+	exit(ret);
+}
+
+static struct wlr_geometry *parse_geometry(const char *str) {
+	// format: {width}x{height}+{x}+{y}
+	if (strlen(str) > 255l) {
```

Comment:
Where does 2551 come from?

Reply:
I misread that today too. That's an L. :) I don't even know if there's a real need for that length check though honestly.

Reply:
Uppercase L perhaps?

Reply:
The L is pretty pointless. There is no reason to force that to be a long type.